src/main/jte/error/known.jte

```diff
@@ -12,7 +12,7 @@
         <div class="max-w-md">
             <h1 class="text-5xl font-bold">${context.localize(error.title)}</h1>
             <p class="py-6">${context.localize(error.message)}</p>
-            <a class="btn btn-primary" href="/">${context.localize("index.go")}</a>
+            <a class="btn btn-primary" href="/" aria-label="${context.localize("index.go")}" title="${context.localize("index.go")}">${context.localize("index.go")}</a>
```

Comment:
This text is already the text of the button, I don't think repeating it is necessary.